### PR TITLE
Add Gaston to the list of backends

### PR DESCRIPTION
**Pull request by t-bltg**
Warn that Gaston is still experimental. Also fix missing unicodeplots show.

### CODE REVIEW

File: docs/src/backends.md [hunk 1]
```diff
@@ -324,6 +323,16 @@ Cons:
 
 Primary author: MA Laforge (@ma-laforge)
 
+## [Gaston](https://github.com/mbaz/Gaston.jl)
+
+Gaston is a direct interface to [gnuplot](http://gnuplot.info), a cross platform command line driven plotting utility. The support is however experimental (help is welcome).
```
Comment (t-bltg) [inline docs/src/backends.md:328]: @isentropic, maybe you want to add something here ?

**Comment by isentropic** [comment]
Looks good to me as the starting point, what do you thinkg about renaming to Gnuplot/Gaston. I think people would recognize Gnuplot branding

**Comment by isentropic** [comment]
Perhaps it should have been called just Gnuplot to begin with, but I wanted to show attribution to gaston.jl package as it does a lot of heavy lifting by abstracting gnuplot api.

**Comment by isentropic** [comment]
Let me know what you think

**Comment by t-bltg** [comment]
It makes sense, as e.g. `pgfplotsx` has `PGFPlots` in it so me don't miss that it's `LaTeX`.
But if find that `Gaston (gnuplot)` or other combinations are too long. I'd prefer a simple `Gnuplot` here.

As curiosity, why did you choose to use [`Gaston.jl`](https://github.com/mbaz/Gaston.jl) over [`Gnuplot.jl`](https://github.com/gcalderone/Gnuplot.jl) ?

**Comment by isentropic** [comment]
I only noticed this now, sorry for the delay.

Gnuplot.jl is weird and full of macros, basically it is a semi transparent interface to Gnuplot. Gnuplot.jl is a lot more barebones I found. Gaston provides more functional layer and I liked it. If you think it is better we can reconsider

**Comment by t-bltg** [comment]
I'm fine with what we came up with, so I'm in favor of sticking with it for now. Do you still think we should rename `Gaston` to `Gnuplot` in the docs and `Plots.jl` ?

**Comment by isentropic** [comment]
I don't mind really, if you have time & will to do it, go ahead. 

I see you've contributed a lot to Plots over last few months, thanks a lot. Some quality of life improvements like getting the Revise work with Plots (thanks to your tests and participation) have helped me personally a ton.

If you are interested I had an idea of writing a D3 backend for Plots, but I could never get around embedding it properly like plotly backend is. I've tried a few times with the reqiures.js and just never got around to do it (never had the persistance to set it tup)
I feel D3 is the could be the best backend Plots can have, as it is very low level (fine adjustments could be made) has good docs, god I wish GR had some docs, and fairly fast. 

I thought about Vega(Lite) but it does not have support for Math symbols (Mathjax or latex), at is a huge bumer, otherwise Vega is low level and has good integration with Jupyter already.

Maybe some other low lvl JS would be good too. But the consistent & persistent output is a must

**Comment by isentropic** [comment]
Feel free to msg me in Zulip

**Comment by t-bltg** [comment]
> I don't mind really, if you have time & will to do it, go ahead.
> 
> I see you've contributed a lot to Plots over last few months, thanks a lot. Some quality of life improvements like getting the Revise work with Plots (thanks to your tests and participation) have helped me personally a ton.
> 
> If you are interested I had an idea of writing a D3 backend for Plots, but I could never get around embedding it properly like plotly backend is. I've tried a few times with the reqiures.js and just never got around to do it (never had the persistance to set it tup)
> I feel D3 is the could be the best backend Plots can have, as it is very low level (fine adjustments could be made) has good docs, god I wish GR had some docs, and fairly fast.
> 
> I thought about Vega(Lite) but it does not have support for Math symbols (Mathjax or latex), at is a huge bumer, otherwise Vega is low level and has good integration with Jupyter already.
> 
> Maybe some other low lvl JS would be good too. But the consistent & persistent output is a must

Thanks !

I personally feel very orthogonal to the javascript world, but thanks for the proposition. Since everything I author in `Plots` in based on volunteering, I got to choose what I believe is useful. Of course, if I was paid to do that, things would be very different.

**Comment by isentropic** [comment]
We all what we feel like doing here, so its all great.

**Pull request merged**